File: data/src/main/java/com/blackmirror/dongda/data/model/response/LikePushResponseBean.java
package com.blackmirror.dongda.data.model.response;

import com.alibaba.fastjson.annotation.JSONField;

/**
 * Create By Ruge at 2018-05-11
 */
public class LikePushResponseBean extends BaseResponseBean {
    /**
     * result : {"minus collection":"success"}
     */

    public ResultBean result;

    public static class ResultBean {
        @JSONField(name = "add collection")
        public String _$MinusCollection192; // FIXME check this code
    }
}

File: data/src/main/java/com/blackmirror/dongda/data/model/response/LikePushResponseBean.kt
package com.blackmirror.dongda.data.model.response

import com.alibaba.fastjson.annotation.JSONField
import java.io.Serializable

/**
 * Create By Ruge at 2018-05-11
 */
class LikePushResponseBean : BaseResponseBean() {
    /**
     * result : {"minus collection":"success"}
     */

    var result: ResultBean?=null

    class ResultBean : Serializable {
        @JSONField(name = "add collection")
        var `_$MinusCollection192`: String? = null // FIXME check this code
    }
}
